File: carsdb/src/main/java/com/example/carsdb/MainActivity.java
package com.example.carsdb;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import java.util.List;

import Data.DataBaseHandler;
import Model.Cars;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        DataBaseHandler dataBaseHandler=new DataBaseHandler(this);
//        dataBaseHandler.addCar(new Cars("Toyota","2500$"));
//        dataBaseHandler.addCar(new Cars("Ferari","20500$"));
//        dataBaseHandler.addCar(new Cars("BMW","4500$"));
//        dataBaseHandler.addCar(new Cars("Opel","500$"));
//        List<Cars> carsList=dataBaseHandler.getAllCars();
//        for (Cars cars:carsList){
Cars cars=dataBaseHandler.getCar(1);
cars.setName("Tesla");
cars.setPrice("15000");
dataBaseHandler.updateCar(cars);
        }
    }

File: app/src/main/java/com/example/kotlinAndroid/MainActivity.kt
package com.example.kotlinAndroid

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun onCreateAge(view: View) {
     var yearsofBreat:Int=editText.text.toString().toInt();
        val carentYear=Calendar.getInstance().get(Calendar.YEAR);
        val myage=carentYear-yearsofBreat;
        tvShow.text="ddd"+myage
    }
}
